int32 anzTiles
image_detecttile_single[] image_detecttile_vector